generators/app/templates/src/app/components/RadioGroup/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+
+import RadioOption from './components/RadioOption';
+import { IRadioOption } from './components/RadioOption/interfaces';
+
+interface Props {
+  disabled?: boolean;
+  className?: string;
+  selectedValue?: string;
+  name?: string;
+  options?: any[];
+  onChange?: (event: React.FormEvent<Element>) => void;
+  [rest: string]: any;
+}
+
+function RadioGroup({
+  disabled = false,
+  selectedValue: selectedValueProp = '',
+  className = '',
+  name = '',
+  options = [],
+  onChange
+}: Props) {
+  const [selectedValue, setSelectedValue] = useState<string>(selectedValueProp);
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSelectedValue(event.target.value);
+
+    if (onChange) {
+      onChange(event);
```

Comment:
Same comment here about the `onChange` and the `selectedValue`.

`selectedValue` isn't being updated by selectedValueProp.